src/Components/QuickGrid/Microsoft.AspNetCore.Components.QuickGrid/test/GridSortTest.cs

```diff
@@ -0,0 +1,182 @@
+        var propertyList = gridSort.ToPropertyList(ascending: true);
+
+        // Assert
+        Assert.Single(propertyList);
+        Assert.Equal("Child.ChildNullableDate", propertyList.First().PropertyName);
+        Assert.Equal(SortDirection.Ascending, propertyList.First().Direction);
+    }
+
+    [Fact]
+    public void ToPropertyName_DescendingSort_ReturnsCorrectDirection()
+    {
+        // Arrange
+        Expression<Func<TestEntity, DateTime?>> expression = x => x.NullableDate;
+
+        // Act
+        var gridSort = GridSort<TestEntity>.ByDescending(expression);
+        var propertyList = gridSort.ToPropertyList(ascending: true);
+
+        // Assert
+        Assert.Single(propertyList);
+        Assert.Equal("NullableDate", propertyList.First().PropertyName);
+        Assert.Equal(SortDirection.Descending, propertyList.First().Direction);
+    }
+
+    [Fact]
+    public void ToPropertyName_MultipleSort_ReturnsAllProperties()
+    {
+        // Arrange
+        Expression<Func<TestEntity, string>> firstExpression = x => x.Name;
+        Expression<Func<TestEntity, DateTime?>> secondExpression = x => x.NullableDate;
+
+        // Act
+        var gridSort = GridSort<TestEntity>.ByAscending(firstExpression)
+            .ThenDescending(secondExpression);
+        var propertyList = gridSort.ToPropertyList(ascending: true);
+
+        // Assert
+        Assert.Equal(2, propertyList.Count);
+        
+        var firstProperty = propertyList.First();
+        Assert.Equal("Name", firstProperty.PropertyName);
+        Assert.Equal(SortDirection.Ascending, firstProperty.Direction);
+        
+        var secondProperty = propertyList.Last();
+        Assert.Equal("NullableDate", secondProperty.PropertyName);
+        Assert.Equal(SortDirection.Descending, secondProperty.Direction);
+    }
+
+    [Fact]
+    public void ToPropertyName_InvalidExpression_ThrowsArgumentException()
+    {
+        // Arrange
+        Expression<Func<TestEntity, string>> invalidExpression = x => x.Name.ToUpper(CultureInfo.InvariantCulture);
+
+        // Act & Assert
+        var gridSort = GridSort<TestEntity>.ByAscending(invalidExpression);
+        var exception = Assert.Throws<ArgumentException>(() => gridSort.ToPropertyList(ascending: true));
+        Assert.Contains("The supplied expression can't be represented as a property name for sorting", exception.Message);
+    }
+
+    [Fact]
+    public void ToPropertyName_MethodCallExpression_ThrowsArgumentException()
+    {
+        // Arrange
+        Expression<Func<TestEntity, string>> invalidExpression = x => x.Name.Substring(0, 1);
+
+        // Act & Assert
+        var gridSort = GridSort<TestEntity>.ByAscending(invalidExpression);
+        var exception = Assert.Throws<ArgumentException>(() => gridSort.ToPropertyList(ascending: true));
+        Assert.Contains("The supplied expression can't be represented as a property name for sorting", exception.Message);
```

Comment:
Can we separate the `Act` and `Assert` sections like below?

```csharp
// Act
var gridSort = GridSort<TestEntity>.ByAscending(invalidExpression);
List<SortedProperty> func() => gridSort.ToPropertyList(ascending: true);

// Assert
var exception = Assert.Throws<ArgumentException>(func);

Assert.Contains(
    "The supplied expression can't be represented as a property name for sorting", 
    exception.Message);
```

See [this](https://github.com/Arnab-Developer/Arc.CustomApp/blob/main/test/Arc.CustomApp.ApplicationTest/Queries/GetStudentByIdQueryHandlerTest.Handle.cs#L33-L38) as an example.

This is applicable for the other areas where you have combined the `Act` and `Assert` sections.